### ISSUE TITLE
App crashes on refreshing in All Courses

### ISSUE THREAD

**Issue by ShridharGoel** [coding-blocks/CBOnlineApp#171]
**Describe the bug**
App crashes on refreshing in All Courses

**To Reproduce**
1. Go to All Courses.
2. Refresh by swiping down.
3. App will crash.

**Expected behavior**
App shouldn't crash and the list should reload.

**Logs**
```
android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
        at android.view.ViewRootImpl.checkThread(ViewRootImpl.java:6958)
        at android.view.ViewRootImpl.invalidateChildInParent(ViewRootImpl.java:1120)
        at android.view.ViewGroup.invalidateChild(ViewGroup.java:5207)
        at android.view.View.invalidateInternal(View.java:13676)
        at android.view.View.invalidate(View.java:13640)
        at android.view.View.invalidateParentIfNeeded(View.java:13829)
        at android.view.View.clearAnimation(View.java:20211)
        at androidx.swiperefreshlayout.widget.SwipeRefreshLayout.a(SwipeRefreshLayout.java:486)
        at androidx.swiperefreshlayout.widget.SwipeRefreshLayout.a(SwipeRefreshLayout.java:472)
        at androidx.swiperefreshlayout.widget.SwipeRefreshLayout.setRefreshing(SwipeRefreshLayout.java:434)
        at com.codingblocks.cbonlineapp.c.b$e$b.a(AllCourseFragment.kt:176)
        at com.codingblocks.cbonlineapp.c.b$e$b.invoke(AllCourseFragment.kt:32)
        at kotlin.c.a$a.run(Thread.kt:30)
```

**GIF**
![ezgif com-video-to-gif (76)](https://user-images.githubusercontent.com/35566748/57866121-6cb92080-781c-11e9-941a-b833f5c363c3.gif)

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

**Comment by ShridharGoel** [comment]
@aggarwalpulkit596 Please add bounty labels.

**Comment by mannprerak2** [comment]
It's not crashing in my device
Can u send a screen recording ?

**Comment by ShridharGoel** [comment]
@mannprerak2 Added in issue description.

**Comment by aggarwalpulkit596** [comment]
Closed via #177
